root/release/edit/tracklist.tt

```diff
@@ -286,6 +286,8 @@
             [%- END %]
           </select>
           (<a href="[% c.uri_for('/doc/Release/Format') %]" target="_blank">[% l('help') %]</a>)
+          <input type="checkbox" data-bind="checked: formatUnknownToUser, enable: formatID() == ''" />
```

Comment:
NB: `disable: formatID` is shorter and also works here, but I felt it was better to be more explicit.